src/Chapter05.Tests/Constants.cs

```diff
@@ -0,0 +1,7 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter05.Tests;
+
+public class Constants
+{
+    public const string Inigo = "Inigo";
+    public const string Montoya = "Montoya";
```

Comment:
Not saying you should do this but there is a fun trick here.
```suggestion
    public const string Inigo = nameof(Inigo);
    public const string Montoya = nameof(Montoya);
```